### PR TITLE
added prerequisite to enable marketplace

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise- 4.11-4.12

JIRA issue:
https://issues.redhat.com/browse/RHDEVDOCS-4561

Preview pages: https://51736--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/installing-openshift-gitops.html

SME+QE review: @sadlerap @wtam2018  @bluengo @vdemeester @sbose78 
Peer-review: @Preeticp @sounix000

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]

**Comment by wtam2018** [comment]
LGTM

**Comment by varshab1210** [comment]
LGTM

**Comment by rjeczkow** [comment]
LGTM

**Comment by Preeticp** [comment]
@sounix000 @vdemeester @sadlerap Debargho has fixed this issue for Pipelines and SBO as well. Request your LGTMs for the same. Thank you.

File: cicd/pipelines/installing-pipelines.adoc [hunk 1]
```diff
@@ -17,6 +17,7 @@ This guide walks cluster administrators through the process of installing the {p
 
 * You have installed `oc` CLI.
 * You have installed xref:../../cli_reference/tkn_cli/installing-tkn.adoc#installing-tkn[OpenShift Pipelines (`tkn`) CLI] on your local system.
+* Your cluster has the link:https://docs.openshift.com/container-platform/4.11/post_installation_configuration/cluster-capabilities.html[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
```
Comment (sounix000) [inline cicd/pipelines/installing-pipelines.adoc:20]: Don't use direct HTML links if the documentation you are pointing to is part of the same repository. Instead, use xref and point towards the correct asciidoc file.

**Review by vdemeester — APPROVED** [review]

**Review by sadlerap — COMMENTED** [review]
LGTM with the fixes suggested by @sounix000.

File: cicd/pipelines/installing-pipelines.adoc (hunk 1 above)
Comment (Preeticp) [reply cicd/pipelines/installing-pipelines.adoc:20]: For your reference the Serverless PR for the same issue: https://github.com/openshift/openshift-docs/pull/51572/files

File: modules/installing-gitops-operator-in-web-console.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@
 * Access to the {product-title} web console.
 * An account with the `cluster-admin` role.
 * You are logged in to the {product-title} cluster as an administrator.
+* Your cluster has the xref:../../post_installation_configuration/cluster-capabilities.adoc#cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
```
Comment (bburt-rh) [inline modules/installing-gitops-operator-in-web-console.adoc:14]: `xref` links are not allowed in module files in the OCP docs repo. Please remove the xref from this module file. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

File: cicd/gitops/installing-openshift-gitops.adoc [hunk 3]
```diff
@@ -9,6 +9,19 @@ toc::[]
 [role="_abstract"]
 {gitops-title} uses Argo CD to manage specific cluster-scoped resources, including cluster Operators, optional Operator Lifecycle Manager (OLM) Operators, and user management.
 
+[discrete]
+== Prerequisites
+
+* Access to the {product-title} web console.
```
Comment (bburt-rh) [inline cicd/gitops/installing-openshift-gitops.adoc:15]:
```suggestion
* You have access to the {product-title} web console.
```
Per the ISG, `Make list items grammatically parallel. For example, do not use a mix of phrases and sentences or a mix of passive and active voice in the same list."

File: cicd/gitops/installing-openshift-gitops.adoc [hunk 4]
```diff
@@ -9,6 +9,19 @@ toc::[]
...
+* An account with the `cluster-admin` role.
```
Comment (bburt-rh) [inline cicd/gitops/installing-openshift-gitops.adoc:16]:
```suggestion
* You are logged in as a user with the `cluster-admin` role.
```
Per the ISG, `Make list items grammatically parallel. For example, do not use a mix of phrases and sentences or a mix of passive and active voice in the same list."

**Pull request merged**